src/main/java/jeryl/fyp/model/student/ProjectStatus.java

```diff
@@ -9,7 +9,7 @@
  */
 public class ProjectStatus {
     public static final String MESSAGE_CONSTRAINTS =
-            "Status should only be of the form YTS, IP OR DONE";
+            "Status should only be of the form YTS, IP or DONE";
```

Comment:
I thought this cannot be changed? Not a bug so

Reply:
oof we can't change this? :((1